src/arrayMethodSort.js

```diff
@@ -5,8 +5,38 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function(compareFunction = stringCompareElements) {
```

And you can remove this if block
![image](https://github.com/mate-academy/js_array-method-sort/assets/80700247/ca6580e1-c3ce-43a0-97a9-6ee4a71bda08)